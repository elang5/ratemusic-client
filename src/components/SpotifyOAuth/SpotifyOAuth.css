.spotify-login > a {
  text-decoration: none;
  color: #191414;
  font-size: 14px;
}

.spotify-login {
  border: none;
  padding: 10px 12px;
  background: #1DB954;
}