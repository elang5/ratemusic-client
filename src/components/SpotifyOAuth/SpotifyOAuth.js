import React, { Component } from 'react'

export class SpotifyOAuth extends Component {
  render() {
    return (
      <div className="container">
        <button>
          <a href="http://localhost:8888">Sign In To Spotify</a>
        </button>
      </div>
    )
  }
}

export default SpotifyOAuth




