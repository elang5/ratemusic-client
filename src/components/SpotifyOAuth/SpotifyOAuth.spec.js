import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import SpotifyOAuth from './SpotifyOAuth'

describe(`SpotifyOAuth Component`, () => {

  it(`renders the SpotifyOAuth component`, () => {
    const wrapper = shallow(<SpotifyOAuth />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})