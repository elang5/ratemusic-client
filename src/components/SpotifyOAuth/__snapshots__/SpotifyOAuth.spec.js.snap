// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpotifyOAuth Component renders the SpotifyOAuth component 1`] = `
<div
  className="container"
>
  <button
    className="spotify-login"
  >
    <a
      href="http://localhost:8888"
    >
      Sign In To Spotify
    </a>
  </button>
</div>
`;
